### ISSUE TITLE
Syntax error in generated code of out parameter's reflection metadata

### ISSUE THREAD

**Issue by yuvaltz** [bridgedotnet/Bridge#1900]
### Expected
Parameter type metadata should not have a trailing `&`

### Actual
out parameter type metadata is

```js
"parameterType":System.Int32&
```
Causing `SyntaxError: Unexpected token`

### Steps To Reproduce
Sample ([Deck](http://deck.net/33196df431eeee7c21b8e9359ff65ed6)):
```csharp
class Program
{
    static void Main(string[] args)
    {
        int value;
        TryGetValue(out value);
        Console.WriteLine(value);
    }
    
    [Bridge.Reflectable(true)]
    private static bool TryGetValue(out int value)
    {
        value = 1;
        return true;
    }
}
```

Generated metadata
```javascript
Bridge.setMetadata(Demo.Program, function () { return {"members":[{"accessibility":1,"name":"TryGetValue","isStatic":true,"type":8,"paramsInfo":[{"name":"value","isOut":true,"parameterType":System.Int32&,"position":0}],"sname":"tryGetValue","returnType":Boolean,"params":[System.Int32&]}]}; });
```

**Comment by geoffreymcgill** [comment]
Thanks for reporting this defect. We will fix right away and try to include in the upcoming 15.2.0 release.

**Comment by geoffreymcgill** [comment]
This defect has been included in the Bridge 15.2.0 release. Now available on [NuGet](https://www.nuget.org/packages/bridge) and [Visual Studio Gallery](https://visualstudiogallery.msdn.microsoft.com/dca5c80f-a0df-4944-8343-9c905db84757).

**Comment by yuvaltz** [comment]
Generic out parameters are still getting a trailing `&`

[Deck](http://deck.net/3dd8cdf8f05c8dcea3b05410dcf6acc8)
```csharp
public class Test
{
    [Reflectable(true)]
    public static void GetValue<T>(out T value)
    {
        value = default(T);
	}
}
```

Output
```
[{"name":"value","isOut":true,"parameterType":T&,"position":0}],"tpcount":1,"sname":"getValue","returnType":Object,"params":[T&]}]}; });
```
